### ISSUE TITLE
Allow user-specified ordering of struct fields

### ISSUE THREAD

**Issue by phiryll** [phiryll/lexy#17]
https://github.com/phiryll/lexy/blob/48c55ee9c5c3662ad489190d24bd52002ac6b93d/README.md?plain=1#L98

- [ ] depends on #11

**Comment by phiryll** [comment]
This is no longer relevant for version 0.1.
Ref #40

**Comment by phiryll** [comment]
There's no good way to complete #40, and therefore no reason for this.
